plugin/src/arch/x64/debug.rs

```diff
@@ -51,8 +58,10 @@ pub fn format_opdata(name: &str, data: &Opdata) -> Vec<String> {
     forms
 }
 
-static REGS: [&str; 16] = ["a",  "d",  "c",   "b",   "bp",  "sp",  "si",  "di",
-                                   "r8", "r9", "r10", "r11", "r12", "r13", "r14", "r15"];
+static REGS: [&str; 16] = [
+    "a", "d", "c", "b", "bp", "sp", "si", "di", "r8", "r9", "r10", "r11", "r12", "r13", "r14",
+    "r15",
+];
```

Comment:
Hmm I guess the vertical alignment was already not perfect here, but rustfmt's take is weird as well.